engineapi/backups.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const (
+	backupStateStarting   = "starting"
```

Comment:
Can we just use these constants from https://github.com/longhorn/backupstore/commit/cc70e06e?

Reply:
The introduction of `starting` state makes the incompatibility of the new `longhorn-manager` and the old `longhorn-engine`. So, checking the engine CLI/API version is necessary. Do we need to include the feature in v1.4.x? 
cc @innobead

Reply:
Where is the incompatible part? I saw all changes are internally for backupstore lib.

Reply:
For any incompatible change, as discussed before, no need, and be unsuitable for backport. 

cc @longhorn/dev

Reply:
https://github.com/longhorn/longhorn-manager/pull/1813#issuecomment-1531327304